PROPERTIES.bzl

```diff
@@ -3,6 +3,7 @@ PROPERTIES = {
     # Version of google-java-format is downgraded from 1.8 to 1.7, because 1.8 supports java 11 minimum, while our JRE is java 8.
     "version.google_java_format": "1.7",
     "version.io_grpc_java": "1.42.1",
+    "commit.gapic_showcase": "90d73532a0cab753a85f45c158394f24fc21d91a",
```

Comment:
We plan to remove this properties file soon and use maven to load dependencies, can we inline the showcase hash?

Reply:
Done